.pylintrc

```diff
@@ -136,7 +136,8 @@ disable=print-statement,
         duplicate-code,
         too-few-public-methods,
         unnecessary-pass,
-        cyclic-import
+        cyclic-import,
+        arguments-differ
```

Comment:
Why are we skipping this now ?

Reply:
Because when you specify several arguments in the forward pass (instead of *args, **kwargs in the nn.Module), you get this warning. It was starting to be everywhere in the code.

Reply:
Ok